src/Explorer/Explorer.ts

```diff
@@ -509,6 +510,14 @@ export default class Explorer {
       return false;
     });
 
+    this.isServerlessEnabled = ko.computed(
+      () =>
+        this.databaseAccount &&
+        this.databaseAccount()?.properties?.capabilities?.find(

```

Comment:
`Array.find` not supported in IE

Reply:
We polyfill it already and use it elsewhere. 

babel-polyfill will polyfill anything that is "polyfillable". We should never avoid methods unless it truly can't be poly filled which is rare.